sim/paladin/items.go

```diff
@@ -227,3 +234,51 @@ func crusadersZealAura465414(character *core.Character) *core.Aura {
 		},
 	})
 }
+
+func libramOfWrathAura(agent core.Agent) {
```

Comment:
Is this shared with something else? If not it doesn't need to be a separate function